cli/config/cliconfig.cpp

```diff
@@ -1,15 +1,29 @@
 #include "cliconfig.h"
 #include "toml/cpptoml.h"
 #include "utils/cfg.h"
+#include "utils/common.h"
 
 #include <iostream>
 
 #define BEGIN  "\x1B[33m"
 #define END  "\x1B[0m"
 #define YELLOW(x) BEGIN x END
 
-CLIConfig::CLIConfig(std::string tomlConfigPath):
-    m_tomlPath(std::move(tomlConfigPath)) {}
+namespace
+{
+inline std::string fileDoesNotContain(std::string const& filePath)
```

Comment:
`formatMessageFileDoesNotContain`?

Reply:
Or, perhaps, even better, define custom errors?

Reply:
Renamed to  `formatMessageFileDoesNotContain`.

>Or, perhaps, even better, define custom errors?
Not really doable, it's pretty specific for this file only.